### PR TITLE
Ensure descending year order on the discover page

### PR DESCRIPTION
**Pull request by kvalev**
reported here https://github.com/kvalev/photoprism/pull/17#issuecomment-1294792011
related to #3

### CODE REVIEW

**Comment by kvalev** [comment]
@fly-man- I couldn't really reproduce it, but this PR should fix the problem for you, as it makes the year order deterministic. The only thing I disagree with in your original comment is the order - I think from latest to oldest year makes more sense. Let me know what you think.

@rickysarraf This PR also contains a couple of minor performance improvements, but I havent really benchmarked it myself. I this something that you can help out with?

**Comment by rickysarraf** [comment]
>@rickysarraf This PR also contains a couple of minor performance improvements, but I havent really benchmarked it myself. I this something that you can help out with?

@kvalev I will pick this MR in the new test build I trigger today.

File: frontend/src/pages/discover/today.vue [hunk 1]
```diff
@@ -105,11 +91,25 @@ export default {
         merged: true,
         month: this.filter.month,
         day: this.filter.day,
+        order: "newest",
```
Comment (rickysarraf) [inline frontend/src/pages/discover/today.vue]: So the results I see with this MR are in line with this code. The `order` that is rendered is `newest`.

**Comment by kvalev** [comment]
I was able to reproduce it in both of the ephemeral Photoprism instances. I will test later today whether this PR fixes the issue:

Screenshot 2023-05-12 at 02-59-06 PhotoPrism Demo

**Pull request merged**